manticore/seth.py

```diff
@@ -823,6 +823,37 @@ def transaction(self, caller, address, value, data):
 
         return status
 
+    def multi_tx_analysis(self, solidity_filename):
+        with open(solidity_filename) as f:
+            source_code = f.read()
+
+        user_account = self.create_account(balance=1000)
+        contract_account = self.solidity_create_contract(source_code, owner=user_account)
+        attacker_account = self.create_account(balance=1000)
+
+        prev_coverage = 0
+        current_coverage = 0
+
+        while current_coverage < 100:
+
+            symbolic_data = self.make_symbolic_buffer(320)
+            symbolic_value = self.make_symbolic_value()
+
+            self.transaction(caller=attacker_account,
+                          address=contract_account,
+                          data=symbolic_data,
+                          value=symbolic_value )
+
+            prev_coverage = current_coverage
+            current_coverage = self.global_coverage(contract_account)
+            found_new_coverage = prev_coverage != current_coverage
+
+            if not found_new_coverage:
```

Comment:
@yan thoughts on this style of the intermediate variable? i wanted to try to make the code self documenting and avoid a comment which could get out of sync with the code.

Reply:
i like it! the only issue (error i made in my pseudocode too) is comparing floating point values with `==`/`!=` can produce false results if the order of operations changes; something like `found_new_coverage = prev_coverage > current_coverage` might be more accurate.